tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "@quasar/app/tsconfig-preset",
   "compilerOptions": {
-    "baseUrl": "."
+    "baseUrl": ".",
+    "lib": ["ES2019", "DOM"]
```

Comment:
dom was needed after switching to es2019; es2019 was needed because of the `.flat()`. Might be worth it to check other aproahces

Reply:
changed the implementation to a `forEach` based solution